### PR TITLE
Add bottom navbar with icons

### PR DESCRIPTION
**Pull request by d-tello**

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -1,21 +1,22 @@
...
+  <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload', defer: true %>
+</head>
 
-    <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload', defer: true %>
-  </head>
+<body>
+  <%= render 'shared/navbar' %>
+  <%= render 'shared/flashes' %>
+  <%= yield %>
+  <%= render 'shared/bottom-navbar' %>
```
Comment (danburck) [inline app/views/layouts/application.html.erb:19]: @d-tello When you add the login page, there probably needs to be a logic that the navbar doesnt show there.

File: app/views/shared/_bottom-navbar.html.erb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<nav class="navbar fixed-bottom navbar-dark bg-white">
+   <%= link_to root_path do %>
```
Comment (danburck) [inline app/views/shared/_bottom-navbar.html.erb:2]: fix indentation

**Review by danburck — CHANGES_REQUESTED** [review]
Made some small comments. Can you fix those? Then its ready to merge

File: app/views/shared/_bottom-navbar.html.erb (hunk 2 above)
Comment (d-tello) [reply app/views/shared/_bottom-navbar.html.erb]: Corrected

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (d-tello) [reply app/views/layouts/application.html.erb:19]: My idea was that was to add a top navbar where the user can go in his profile page (and eventually log out) and another icon for the messages.
In this way we should also blacklist the home page and make it available only if the user is authenticated, like for example the facebook or instagram app

Comment (danburck) [reply app/views/layouts/application.html.erb:19]: @d-tello yup, sound good. Many apps have 2-3 icons on the top right corner for forther settings.

Comment (d-tello) [reply app/views/layouts/application.html.erb:19]: @danburck I added it in the PR for the top navbar

**Pull request merged**